### PR TITLE
fix typos in new Chocolatey feature 'choco export'

### PR DESCRIPTION
**Pull request by mwallner**

### CODE REVIEW

File: src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs [hunk 1]
```diff
@@ -103,7 +103,7 @@ choco export
  - 0: operation was successful, no issues detected
  - -1 or 1: an error has occurred
 
-If you find other exit codes that we have not yet documented, please
+If you find other exit codes that we have not yet been documented, please
```
Comment (gep13) [inline src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs]: @mwallner can you remove this change, as it doesn't actually make sense the way you have written it.  Once changed, we can get this merged in.

**Review by gep13 — CHANGES_REQUESTED** [review]
Small change before merging, thanks!

**Comment by gep13** [comment]
@mwallner thanks for closing this off.  It was easier to redo the commit locally during the last stream, but I attributed you on the commit, so you will get some "internet points" for your help 😄

**Comment by mwallner** [comment]
:-D that wouldn't be neccessary, but I do appreciate it ;-)

**Pull request closed without merging**